screen/model/media/imagehandler.js

```diff
@@ -0,0 +1,24 @@
+
+const MediaHandler = require('./mediahandler');
+
+module.exports = class ImageHandler extends MediaHandler {
+
+    constructor(id, dom){
+        super(id, dom);
+    }
+
+    init(createMessage){
+        this.uiWrapper.innerHTML = `<img id = 'image-${this.id}' class = 'image-media' src = '${createMessage.resource}'>`; 
+    }
+
+    handleMessage(msg){
+        if (!msg.command){
```

Comment:
Is this check needed? If you remove it, the message will end up in MediaHandler's default case. (Right?)